ArcadiaParties.Data/Helpers/AutoMapperProfiles.cs

```diff
@@ -0,0 +1,17 @@
+﻿using ArcadiaParties.Data.Abstractions.DTOs;
+using ArcadiaParties.Data.Models;
+using AutoMapper;
+using System.Linq;
+
+namespace ArcadiaParties.Data.Helpers
+{
+    class AutoMapperProfiles : Profile
+    {
+        public AutoMapperProfiles()
+        {
+            CreateMap<User, UserDTO>()
+                .ForMember(uDTO => uDTO.UserRoles, opt => opt.MapFrom(x => x.UserRoles.Select(y => y.Role.Name).ToList()))
```

Comment:
Why `opt`?

Reply:
Cause parameter name is `memberOptions`. I've renamed it for particular situation

Reply:
Ok.